plugins/source/aws/resources/services/ssoadmin/instances_fetch.go

```diff
@@ -0,0 +1,35 @@
+package ssoadmin
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/ssoadmin"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+func fetchSsoadminInstances(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- interface{}) error {
+	fmt.Print("*******FETCH********\n")
```

Comment:
Forgotten print stmt, also two more below on line 20-21

Reply:
Fixed, thanks. Wasn't forgotten at the time I committed it, was demonstrating a problem (note the WIP title at the time)